test/sample_unit_test.c

```diff
@@ -0,0 +1,16 @@
+#include <criterion/criterion.h>
+#include <stdio.h>
+
+void setup(void)
+{
```

Comment:
Maybe put the curly braces on the same line as the function prototype, for consistency's sake

Reply:
Ouh yes, that was a copy-pasta from Criterion's example, will fix that